code/modules/reagents/reagent_containers/cups/drinkingglass.dm

```diff
@@ -17,6 +17,7 @@
 	custom_price = PAYCHECK_LOWER
 	//the screwdriver cocktail can make a drinking glass into the world's worst screwdriver. beautiful.
 	toolspeed = 25
+	usesound = list('sound/items/screwdriver.ogg', 'sound/items/screwdriver2.ogg')
```

Comment:
If you don't mind, can you take these off of the base definition and only set them when you set it to screwdriver?

Reply:
Yeah hold on, gimme a bit to swap repos and test it.
Shouldn't be an issue from what I've seen, but might as well.

Reply:
Shooooould be done.